### ISSUE TITLE
project

### ISSUE THREAD

**Issue by shanow7** [Double2Sky/OffenOsint#2]
when will be ready?

**Comment by Double2Sky** [comment]
Already done.

**Comment by shanow7** [comment]
I've problem with import

Result code (RC): | E_INVALIDARG (0x80070057)

can you help?

**Comment by shanow7** [comment]
I've problem with redis-stable on iKy

(osint㉿OSINT)-[~/OsintTools/iKy/redis-stable]
└─$ sudo make install
cd src && make install
make[1]: Entering directory '/home/osint/OsintTools/iKy/redis-stable/src'
    CC Makefile.dep

Hint: It's a good idea to run 'make test' ;)

    INSTALL install
    INSTALL install
    INSTALL install
    INSTALL install
    INSTALL install
make[1]: Leaving directory '/home/osint/OsintTools/iKy/redis-stable/src'
                                                                                                       
┌──(osint㉿OSINT)-[~/OsintTools/iKy/redis-stable]
└─$ make test
cd src && make test
make[1]: Entering directory '/home/osint/OsintTools/iKy/redis-stable/src'
You need tcl 8.5 or newer in order to run the Redis test
make[1]: *** [Makefile:349: test] Error 1
make[1]: Leaving directory '/home/osint/OsintTools/iKy/redis-stable/src'
make: *** [Makefile:6: test] Error 2

can you help?

**Comment by shanow7** [comment]
most of the programs do not work properly, did you do the tests?

![image](https://user-images.githubusercontent.com/37192410/105643365-b41bff00-5e8f-11eb-8d4e-df5a4ec45cef.png)

![image](https://user-images.githubusercontent.com/37192410/105643469-4c19e880-5e90-11eb-926f-f6e06864412d.png)

**Comment by Double2Sky** [comment]
> I've problem with redis-stable on iKy
> 
> (osint㉿OSINT)-[~/OsintTools/iKy/redis-stable]
> └─$ sudo make install
> cd src && make install
> make[1]: Entering directory '/home/osint/OsintTools/iKy/redis-stable/src'
> CC Makefile.dep
> 
> Hint: It's a good idea to run 'make test' ;)
> 
> ```
> INSTALL install
> INSTALL install
> INSTALL install
> INSTALL install
> INSTALL install
> ```
> 
> make[1]: Leaving directory '/home/osint/OsintTools/iKy/redis-stable/src'
> 
> ┌──(osint㉿OSINT)-[~/OsintTools/iKy/redis-stable]
> └─$ make test
> cd src && make test
> make[1]: Entering directory '/home/osint/OsintTools/iKy/redis-stable/src'
> You need tcl 8.5 or newer in order to run the Redis test
> make[1]: *** [Makefile:349: test] Error 1
> make[1]: Leaving directory '/home/osint/OsintTools/iKy/redis-stable/src'
> make: *** [Makefile:6: test] Error 2
> 
> can you help?

$ cd /home/osint/OsintTools/iKy
$ docker-compose up --build

**Comment by shanow7** [comment]
┌──(osint㉿OSINT)-[~]
└─$ cd /home/osint/OsintTools/iKy
                                                                                                     
┌──(osint㉿OSINT)-[~/OsintTools/iKy]
└─$ docker-compose up --build
/usr/local/lib/python3.9/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.22) or chardet (4.0.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Creating network "iky_default" with the default driver
ERROR: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network

**Comment by Double2Sky** [comment]
> ERROR: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network

You must disable the VPN

**Comment by shanow7** [comment]
/usr/local/lib/python3.9/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.22) or chardet (4.0.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Builds, (re)creates, starts, and attaches to containers for a service.

Unless they are already running, this command also starts any linked services.

The `docker-compose up` command aggregates the output of each container. When
the command exits, all containers are stopped. Running `docker-compose up -d`
starts the containers in the background and leaves them running.

If there are existing containers for a service, and the service's configuration
or image was changed after the container's creation, `docker-compose up` picks
up the changes by stopping and recreating the containers (preserving mounted
volumes). To prevent Compose from picking up changes, use the `--no-recreate`
flag.

If you want to force Compose to stop and recreate all containers, use the
`--force-recreate` flag.

Usage: up [options] [--scale SERVICE=NUM...] [SERVICE...]

Options:
    -d, --detach               Detached mode: Run containers in the background,
                               print new container names. Incompatible with
                               --abort-on-container-exit.
    --no-color                 Produce monochrome output.
    --quiet-pull               Pull without printing progress information
    --no-deps                  Don't start linked services.
    --force-recreate           Recreate containers even if their configuration
                               and image haven't changed.
    --always-recreate-deps     Recreate dependent containers.
                               Incompatible with --no-recreate.
    --no-recreate              If containers already exist, don't recreate
                               them. Incompatible with --force-recreate and -V.
    --no-build                 Don't build an image, even if it's missing.
    --no-start                 Don't start the services after creating them.
    --build                    Build images before starting containers.
    --abort-on-container-exit  Stops all containers if any container was
                               stopped. Incompatible with -d.
    -t, --timeout TIMEOUT      Use this timeout in seconds for container
                               shutdown when attached or when containers are
                               already running. (default: 10)
    -V, --renew-anon-volumes   Recreate anonymous volumes instead of retrieving
                               data from the previous containers.
    --remove-orphans           Remove containers for services not defined
                               in the Compose file.
    --exit-code-from SERVICE   Return the exit code of the selected service
                               container. Implies --abort-on-container-exit.
    --scale SERVICE=NUM        Scale SERVICE to NUM instances. Overrides the
                               `scale` setting in the Compose file if present.

what's next?

**Comment by Double2Sky** [comment]
> what's next?
$ systemctl start docker
$ cd /home/osint/Osinttools/iKy
$ docker-compose up

**Comment by shanow7** [comment]
can you add sifter, Sn1per,discover, OSINT-SAN, getcontact and KaliIntelligenceSuite

**Comment by Double2Sky** [comment]
> can you add sifter, Sn1per,discover, OSINT-SAN, getcontact and KaliIntelligenceSuite

You know how to install discover and sniper on the same system? There is a file conflict in discover.sh which causes discover to refuse to start.

**Comment by shanow7** [comment]
can you add https://github.com/Datalux/Osintgram

**Comment by mattwilson21** [comment]
This is a cool VM thanks

**Comment by mattwilson21** [comment]
This is still a great OSINT VM - most tools work but this project could use some updating.     TOR can't be run as root anymore.
